spec/features/editing_tags/edit_tags_spec.rb

```diff
@@ -62,14 +63,17 @@ def then_i_can_see_the_tags
       expect(page).to have_content("Tag to select 2")
       expect(page).not_to have_content("Initial tag")
     end
+  end
 
+  def and_i_see_the_timeline_entry
+    click_on "Document history"
     within first(".app-timeline-entry") do
       expect(page).to have_content I18n.t!("documents.history.entry_types.updated_tags")
     end
   end
 
   def and_the_preview_creation_succeeded
-    expect(@request).to have_been_requested
+    expect(@put_content_request).to have_been_requested
     expect(page).to have_content(I18n.t!("user_facing_states.draft.name"))
 
     expect(a_request(:put, /content/).with { |req|
```

Comment:
Hmm this seems to be another double one - am I missing something and that these are testing two different things? I'm now a bit confused since they're quite prevalent.